Adyen.Test/Adyen.Test.csproj

```diff
@@ -6,15 +6,15 @@
 
     <IsPackable>false</IsPackable>
 
-    <AssemblyVersion>11.2.1</AssemblyVersion>
+    <AssemblyVersion>12.0.0</AssemblyVersion>
```

Comment:
```suggestion
    <AssemblyVersion>12.0.0-beta</AssemblyVersion>
```

Reply:
`AssemblyVersion` stricly adheres to X.X.X format, it cannot build the package otherwise. For the previous beta we did this the same way